.teamcity/_self/projects/WebApp.kt

```diff
@@ -107,9 +101,8 @@ object RunCalypsoE2eDesktopTests : BuildType({
 				export BROWSERSIZE="desktop"
 				export BROWSERLOCALE="en"
 				export NODE_CONFIG="{\"calypsoBaseURL\":\"${'$'}{URL%/}\"}"
-				export TEST_FILES=${'$'}(join ',' ${'$'}(find specs*/**/*spec.js -type f -not -path specs-playwright/*))
 
-				yarn magellan --config=magellan-calypso.json --max_workers=%E2E_WORKERS% --suiteTag=parallel --local_browser=chrome --mocha_args="--reporter mocha-multi-reporters --reporter-options configFile=mocha-reporter.json" --test=${'$'}{TEST_FILES}
+				yarn magellan --config=magellan-calypso.json --max_workers=%E2E_WORKERS% --local_browser=chrome --mocha_args="--reporter mocha-multi-reporters --reporter-options configFile=mocha-reporter.json"
```

Comment:
Instead of passing a list of files using `--test=...`, rely on the property `mocha_tests` in `magellan-calypso.json`.

The effect of this change is that now `test/e2e/specs/specs-calypso/wp-calypso-seo-preview.js` is included.

Reply:
It follows naturally then that we never ran the SEO spec prior to this, eh?

Reply:
😬